docs/develop/loading-the-dom-and-runtime-environment.md

```diff
@@ -58,16 +58,29 @@ The following events occur when an Outlook add-in starts:
     
 5. The browser control loads the DOM and HTML body, and calls the event handler for the  **onload** event.
     
-6. Outlook calls the event handler for the [initialize](https://docs.microsoft.com/javascript/api/office) event of the [Office](https://docs.microsoft.com/javascript/api/office) object of the add-in.
+6. Outlook loads the runtime environment, which downloads and caches the JavaScript API for JavaScript library files from the content distribution network (CDN) server, and then calls the event handler for the [initialize](https://docs.microsoft.com/javascript/api/office?view=office-js) event of the [Office](https://docs.microsoft.com/javascript/api/office?view=office-js) object of the add-in, if a handler has been assigned to it. At this time it also checks to see if any callbacks (or chained `then()` functions) have been passed (or chained) to the `Office.onReady` handler. For more information about the distinction between `Office.initialize` and `Office.onReady`, see [Initializing your add-in](/office/dev/add-ins/develop/understanding-the-javascript-api-for-office#initializing-your-add-in).

```

Comment:
Suggest updating URLs as follows:

- change the target URL for `initialize` hyperlink to:  `/javascript/api/office#initialize-reason-`
- change the target URL for `Office` hyperlink to: `/javascript/api/office`